### ISSUE TITLE
`require-description` shouldn't require a description for `eslint-enable`

### ISSUE THREAD

**Issue by DetachHead** [mysticatea/eslint-plugin-eslint-comments#61]
when using `eslint-enable`, there's already an `eslint-disable` comment with the description on it so there's no reason to provide another description when re-enabling it.

```ts
describe('any', () => {
    /* eslint-disable @typescript-eslint/no-explicit-any -- testing the any type */
    test('pass', () => {
        exactly<any>()(any)
    })
    test('fail', () => {
        // @ts-expect-error doesn't match
        exactly<any>()(number)
        // @ts-expect-error doesn't match
        exactly<number>()(any)
    })
    /* eslint-enable @typescript-eslint/no-explicit-any -- testing the any type */
})

**Comment by MichaelDeBoey** [comment]
Hi @DetachHead!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-eslint-comments

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org

**Comment by DetachHead** [comment]
https://github.com/eslint-community/eslint-plugin-eslint-comments/issues/131
